controllers/userController.js

```diff
@@ -21,14 +21,17 @@ const signup = async (req, res) => {
   }
 };
 
-const login = async (res, req) => {
+const login = async (req, res) => {
   try {
     const { email, password } = req.body;
     if ((!email, !password)) {
       return res.status(400).json({ message: "KEY_ERROR" });
     }
-    await userService.login(email, password);
-    return res.status(200).json({ message: "SUCCESS_LOGIN" });
+    const accessToken = await userService.login(email, password);
+    if (!accessToken) {
+      return res.status(400).json({ message: "NOT_EXIST_TOKEN" });
+    }
+    return res.status(200).json({ accessToken: accessToken });
```

Comment:
```suggestion
    return res.status(200).json({ accessToken });
```
javascript에서는 object의 property name과 value에 담기는 변수명이 똑같으면 위와같이 축약해서 쓸 수 있습니다.